doc/src/getting-started-guide.md

```diff
@@ -365,75 +448,74 @@ Use the `-h` option to get command line help:
 
         $ /usr/local/lib/atomvm/AtomVM /path/to/my/application.avm /path/to/atomvmlib.avm
 
-Currently, the `atomvm` command and libraries must be built and installed from source.
+## Getting Started with AtomVM WebAssembly
 
-> See the AtomVM [Build Instructions](./build-instructions.md) for instructions about how to build AtomVM on the Generic UNIX platform.
+You can deploy AtomVM for WebAssmbly on NodeJS or for common browsers (Safari, Chrome and Chrome-based, Firefox).
```

Comment:
Typo, stray `or` ...NodeJS >or< for common...

Reply:
I think it’s right — you can deploy on node or run in the browser.  Is the  language misleading or could it be said better?

Reply:
No you are right, I think I was just misreading.

Reply:
It is a little confusing actually, because even if it is deployed to NodeJS, you still need to access that with a browser... (otherwise isn't it a tree falling in an empty forest?) Maybe...

> You can deploy AtomVM for WebAssmbly on NodeJS or run locally in common browsers (Safari, Chrome and Chrome-based, Firefox).

Reply:
@pguyot can you weigh in on this?  I haven’t tried either of these, and honestly I didn’t really do a thorough review or re-write of this section.

Reply:
Adopted @pguyot 's language